tests/console/supportutils.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test is files created by supportconfig are readable and contain some basic data.
+# Maintainer: Juraj Hura <jhura@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run "rm -rf nts_* ||:";
+    assert_script_run "supportconfig -t . -B test", 300;
+    assert_script_run  "cd nts_test";
+
+    # Check few file whether expected content is there.
+    assert_script_run  "diff <(awk '/\\/proc\\/cmdline/{getline; print}' boot.txt) /proc/cmdline";
+    assert_script_run  "grep -f /etc/os-release basic-environment.txt > /dev/null";
```

Comment:
why "`>/dev/null`"

Reply:
... what about `grep -q ....`

Reply:
because I don't need the output to be pasted into console, just to make sure it's there

Reply:
> ... what about `grep -q ....`

yeah, guess that's smarter. thx